src/generate/index.js

```diff
@@ -48,7 +64,7 @@ function generateContributorsList(options, contributors) {
     _.map(formatLine),
     _.join('</tr><tr>'),
     newContent => {
-      return `\n<table cellspacing="0" cellpadding="1"><tr>${newContent}</tr></table>\n`
+      return `\n<table><tbody><tr>${newContent}</tr>\n${tableFooter}\n</tbody></table>\n`
```

Comment:
It would make sense to use `<tfoot>` for the `tableFooter` like so:
```diff,js
- `\n<table><tbody><tr>${newContent}</tr>\n${tableFooter}\n</tbody></table>\n`
+ `\n<table><tbody><tr>${newContent}</tr></tbody><tfoot>${tableFooter}</tfoot></table>\n`
```

Reply:
@Berkmann18 done